package.json

```diff
@@ -51,7 +51,7 @@
   "author": "Zed Industries",
   "license": "Apache-2.0",
   "dependencies": {
-    "@agentclientprotocol/sdk": "0.8.0",
+    "@agentclientprotocol/sdk": "github:SteffenDE/acp-typescript-sdk#sd-session-fork-dist",
```

Comment:
Only necessary until https://github.com/agentclientprotocol/typescript-sdk/pull/37 is merged.